config/rbac/controller_roles.yaml

```diff
@@ -0,0 +1,97 @@
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: manager-role
```

Comment:
Probably need to change it to something specific no?

Reply:
This file was a copy of internal/controller/config/rbac/role.yaml. I've removed the temporary file and I've changed the name to assisted-service-manager-role in the original.